x/concentrated-liquidity/pool_test.go

```diff
@@ -163,7 +163,7 @@ func (s *KeeperTestSuite) TestCalcOutAmtGivenIn() {
 				// create second position parameters
 				newLowerPrice := sdk.NewDec(5500)
 				s.Require().NoError(err)
-				newLowerTick := cl.PriceToTick(newLowerPrice) // 84222
+				newLowerTick := cl.PriceToTick(newLowerPrice) // 86129
```

Comment:
What's the reasoning behind this change?

Reply:
Was previously just a copy paste error, 5500 should correspond to tick 86129 (this is only a change in comment, so no logic here was touched)